libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4619,7 +4627,7 @@ uint64_t GCS_MAVLINK::capabilities() const
         // Copter and Sub may also set this bit as they can always terminate
         ret |= MAV_PROTOCOL_CAPABILITY_FLIGHT_TERMINATION;
     }
-
+    ret |= MAV_PROTOCOL_CAPABILITY_FTP;
```

Comment:
we should have a HAL_WITH_MAVFTP define in GCS.h, which should default to !HAL_MINIMIZE_FEATURES
When we add new features we really need to be careful not to build them on boards with HAL_MINIMIZE_FEATURES or we will run out of flash on those boards